dss-collection-nft/contracts/DSSCollection.cdc

```diff
@@ -0,0 +1,626 @@
+/*
+    DSSCollection contains collection group & completion functionality for DSS.
+    Author: Jeremy Ahrens jer.ahrens@dapperlabs.com
+*/
+
+import NonFungibleToken from "./NonFungibleToken.cdc"
+import MetadataViews from 0xMETADATAVIEWSADDRESS
+
+// The DSSCollection contract
+//
+pub contract DSSCollection: NonFungibleToken {
+
+    // Contract Events
+    //
+    pub event ContractInitialized()
+
+    // NFT Collection Events
+    //
+    pub event Withdraw(id: UInt64, from: Address?)
+    pub event Deposit(id: UInt64, to: Address?)
+
+    // Events
+    //
+    pub event CollectionGroupCreated(
+        id: UInt64,
+        name: String,
+        typeName: String,
```

Comment:
we can make this a cadence type

Reply:
Ryan asked for a string, else that is fine with me. It's easier to ingest in dss-core, I understand.